Cargo.toml

```diff
@@ -21,6 +21,8 @@ log = "0.4"
 serde_json = "1.0"
 thiserror = "2.0.11"
 tokio = { version = "1", features = ["full"] }
+payjoin = { version = "0.23.0", features = ["v1", "v2", "io"] }
+minreq = { version = "2.13.2", features = ["https"] }
```

Comment:
If you're using the `io` feature, you're already depending on `reqwest`, so minreq is a second HTTP client. If you want to use minreq to reduce the dependency burden I recommend also writing the `payjoin/io` feature functions using `minreq`.